test/integration/fixtures/publishing_api_payload.json

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
I would avoid using fixtures, as they will get out of date (and you won't know). You can generate a random, valid, payload using the `govuk_schemas` gem:

```
GovukSchemas::RandomExample.for_schema(notification_schema: "specialist_document")
```

http://rubydoc.info/gems/govuk_schemas